src/Jellyfin.Maui/Pages/Facades/BaseContentIdPage.cs

```diff
@@ -18,12 +21,36 @@ protected BaseContentIdPage(TViewModel viewModel)
     {
     }
 
+    /// <summary>
+    /// Gets or sets the selected Args.
+    /// </summary>
+    public string? ItemId
```

Comment:
Why not have this be a `Guid?` parameter?

Reply:
Because it did not work: System.InvalidCastException: 'Object must implement IConvertible.'